packages/integrations/solid/src/index.ts

```diff
@@ -1,30 +1,12 @@
 import type { AstroConfig, AstroIntegration, AstroRenderer } from 'astro';
+import solid, { type Options as ViteSolidPluginOptions } from 'vite-plugin-solid';
 import { getSolidPkgsConfig } from './dependencies.js';
```

Comment:
We should be able to remove `getSolidPkgsConfig` and it's related dep (`vitefu`) since `vite-plugin-solid` also does the same internally. I brought over the logic from it initially.

Reply:
I think something was failing without this, but I'll give it another try.